app/schema.graphql

```diff
@@ -558,6 +581,14 @@ type SpanEdge {
   cursor: String!
 }
 
+type SpanEvaluation implements Evaluation {
+  name: String!
+  score: Float
+  label: String
+  explanation: String
+  spanId: String!
```

Comment:
Same as above - maybe this was for troubleshooting but not needed I think?

Reply:
yup. will remove. thx